tests/async-unit/test_appeal_journey_core.py

```
@@ -728,22 +728,406 @@ def test_review_scenario_stale_key_after_old_pod_edit_is_repaired_end_to_end(sel
+
+    build = _build_script()
+    rollout_at = build.index("# ROLLOUT GATE")
+    backfill_at = build.index('if [ "$SKIP_BACKFILL" = true ]')
+    assert rollout_at < backfill_at
+    # The rollout gate answers only to --skip-journey-gates.
+    rollout_block = build[rollout_at:backfill_at]
+    assert "SKIP_JOURNEY_GATES" in rollout_block
+    assert "SKIP_BACKFILL" not in rollout_block
+
+    root = pathlib.Path(__file__).resolve().parents[2]
+    helped = subprocess.run(
```

Ruff (0.16.3)

[error] `subprocess` call: check for execution of untrusted input

(S603)